ios/FluentUI/Avatar/AvatarView.swift

```diff
@@ -258,6 +258,12 @@ open class AvatarView: UIView {
         }
     }
 
+    @objc open var shouldCalculateRingColor: Bool = false {
+        didSet {
+            updateBorderColor()
```

Comment:
should we only update border when variable value has changed?

Reply:
Could you clarify? I think updateBorderColor is also called whenever the border needs to be updated

Reply:
if oldValue != shouldCaculdateRingColor { updatebordercolor()}

Reply:
Oh sorry okay thank you